binstar_client/utils/projects/uploader.py

```diff
@@ -30,13 +30,17 @@ def stage(self):
             self.domain, self.username, self.project.name)
         data, headers = jencode(self.project.to_stage())
         res = self.session.post(url, data=data, headers=headers)
+        self._check_response(res)
         return res
 
     def commit(self, revision_id):
-        url = "{}/apps/{}/projects/{}/commit".format(
-            self.domain, self.username, self.project.name)
-        data, headers = jencode({'revision_id': revision_id})
+        url = "{}/apps/{}/projects/{}/commit/{}".format(
+            self.domain, self.username,
+            self.project.name, revision_id
+        )
+        data, headers = jencode({})
         res = self.session.post(url, data=data, headers=headers)
+        self._check_response(res, [201])
```

Comment:
I'm not sure that there are any tests for this because there was nothing even checking the http response code